### PR TITLE
Simple readme file

### PR DESCRIPTION
**Pull request by fwenzel**
r? @tofumatt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Phonebook
+Mozilla's internal phonebook app.
+
+## Installation
+* Clone, host on a local, PHP-enabled server, run.
+* Optional: copy ``config-local.php-dist`` to ``config-local.php`` and tweak as needed
```
Comment (tofumatt) [inline README.md]: Technically you only need <code>`</code>.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+*Note:* The app requires LDAP server access. You probably need a VPN connection up and running.
```
Comment (tofumatt) [inline README.md]: Link to VPN mana/whatever page detailing how to connect would be good here; I actually forget which VPN we're even supposed to use for this anymore (IIRC office VPNs were decommissioned, but then spun back up for folks who needed them for things--which one do I use?).

File: README.md (hunk 1 above)
Comment (tofumatt) [inline README.md]: Also a copy+pastable `cp config-local.php-dist config-local.php` is always good ^_^

File: README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Contributing
+
+If you feel so inclined, feel free to contribute:
```
Comment (tofumatt) [inline README.md]: "Feel" twice in a sentence, also not too straightforward for ESL folks :wink: 

`To contribute, file bugs in Bugzilla and use GitHub for code review:`

**Comment by tofumatt** [comment]
Reviewed! r- but already better than the lack of README before. Some tweaks (especially around VPN, maybe mention “without VPN you cannot current develop on phonebook”) and it’s good!

**Comment by fwenzel** [comment]
Thanks for the thorough review, will fix

File: README.md (hunk 3 above)
Comment (fwenzel) [inline README.md]: eew yes terrible style, oops

**Comment by fwenzel** [comment]
Updated, squashed. @tofumatt ^^

**Comment by tofumatt** [comment]
r+ from me! @lonnen, can you have a look/merge? I don't have commit access to this, though I'd be happy to manage the PRs on it if you want to add me.

**Pull request merged**